actors/builtin/market/market_actor.go

```diff
@@ -251,7 +251,7 @@ func (a Actor) PublishStorageDeals(rt Runtime, params *PublishStorageDealsParams
 }
 
 type VerifyDealsForActivationParams struct {
-	DealIDs      []abi.DealID
+	DealIDs      []abi.DealID // PARCHECK: unbounded
```

Comment:
These ones are called by actor code, so we need to check the call sites will respect the bound.